embedded/store/immustore_test.go

```diff
@@ -4253,7 +4253,8 @@ func TestImmudbStoreTruncateUptoTx_WithMultipleIOConcurrency(t *testing.T) {
 	opts := DefaultOptions().
 		WithFileSize(6).
 		WithMaxConcurrency(100).
-		WithMaxIOConcurrency(5)
+		WithMaxIOConcurrency(5).
+		WithVLogCacheSize(0)
```

Comment:
better to disable the vlogCache by default, only enable at specific test cases

Reply:
done